os/arch/arm/src/amebad/Kconfig

```diff
@@ -51,6 +51,55 @@ config RTL8721D_SERIAL_FIFO
 	bool "UART SERIAL FIFO"
 	default n
 
+config AMEBAD_I2S
+	bool "Enable Amebad I2S"
+	default n
+
+if AMEBAD_I2S
+config AMEBAD_I2S_RX
+	bool "Enable I2S RX"
+	default n
+
+config AMEBAD_I2S_TX
+	bool "Enable I2S TX"
+	default n
+
+config AMEBAD_I2S_16BIT_PADDING
+	bool "Enable I2S 16-bit padding"
+	default n
+	---help---
+		Enable 16-bit padding to send 16bit I2S data in 32bit word length
+endif
+
+config AMEBAD_I2C_DYNTIMEO
+	bool "[Amebad I2C]Use dynamic timeouts"
+	default n
+
+config AMEBAD_I2C_DYNTIMEO_USECPERBYTE
+	int "[Amebad I2C]Timeout Microseconds per Byte"
```

Comment:
Is it necessary to mention `[Amebad I2C]` for every config option?

Reply:
Hi, originally it was meant to indicate that the option is for I2C
How about removing `[Amebad I2C]` and adding `comment "I2C Options"` above the I2C configs, will that be better?